l10n_br_cnpj_search_crm/models/l10n_br_base_party_mixin.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2022 KMEE
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from erpbrasil.base.misc import punctuation_rm
+from requests import get
+
+from odoo import _, models
+from odoo.exceptions import UserError
+
+
+class PartyMixinCRM(models.AbstractModel):
+    _inherit = "l10n_br_base.party.mixin"
+
+    def search_cnpj(self):
+        """Search CNPJ by the chosen API"""
```

Comment:
Essa função está identica a versão que está no módulo base. Por que não utilizar o conceito de heranças nesse caso e só alterar o que for realmente necessário para o módulo? Se não, fica irrelevante ter algo assim.